extendeddatasquare.go

```diff
@@ -51,6 +51,17 @@ func ComputeExtendedDataSquare(
 	data [][]byte,
 	codec Codec,
 	treeCreatorFn TreeConstructorFn,
+) (*ExtendedDataSquare, error) {
+	return ComputeExtendedDataSquareLimitParallelOps(data, codec, treeCreatorFn, 0)
```

Comment:
maybe mention what passing 0 does here as a comment

is this the same as passing 1?